pkgs/applications/audio/flac2all/default.nix

```
@@ -0,0 +1,37 @@
+{ python3Packages, lib, flac, lame, opusTools, vorbis-tools, ffmpeg }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "flac2all";
+  version = "5.1";
+
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "OBjlr7cbSx2WOIfZUNwHy5Hpb2Fmh3vmZdc70JiWsiI=";
+  };
+
+  # Not sure why this is needed, but setup.py expects this to be set
+  prePatch = ''
+    echo ${version} > ./flac2all_pkg/version
+  '';
+
+  propagatedBuildInputs = [
+    # Hard requirements
+    python3Packages.pyzmq
+    flac
+    lame
+    # Optional deps
+    opusTools
+    vorbis-tools
+    ffmpeg
+  ];
+
+  # Untested
```

```suggestion
  # has no tests
```

Can we add a pythonImportsCheck instead?

---

It feels a bit awkward, but does work